zipline/assets/assets.py

```diff
@@ -283,21 +396,21 @@ def lookup_symbol(self, symbol, as_of_date, fuzzy=False):
                 if len(fuzzy_sids) == 1:
                     return self._retrieve_equity(fuzzy_sids[0]['sid'])
 
-            sids = sa.select((equities_cols.sid,)).where(
-                (equities_cols.company_symbol == company_symbol) &
-                (equities_cols.share_class_symbol == share_class_symbol)
-            ).execute().fetchall()
+            sids = list(imap(
+                itemgetter('sid'),
```

Comment:
Given that this is all about performance, any idea how this compares to a list comprehension?
```python
sids = [row['sid'] for row in sa.select(...).execute().fetchall()]
```

(Same question below)